### ISSUE TITLE
Issues with Zig

### ISSUE THREAD

**Issue by alosarjos** [rafamadriz/friendly-snippets#520]
The issue I have reported on LazyVim: https://github.com/LazyVim/LazyVim/issues/4746

Trying to figure out where is the upstream issue, I see the error messages stop showing up when I disable `friendly_snippets` from `garymjr/nvim-snippets` so I'm guessing the issue may come from here?

**Comment by alosarjos** [comment]
I have tried applying the patch provided in this PR and seems to fix it: https://github.com/rafamadriz/friendly-snippets/pull/517

**Comment by FObersteiner** [comment]
Multiple PRs tried to address this actually, none of them got merged yet. The earliest I know of is #507.

**Comment by OkelleyDevelopment** [comment]
Hi @alosarjos, we've merged in PR #531 which I've tested and seems to resolve the issues with Zig snippets not loading. Please pull the latest changes from `main` and test again. 

Apologies a fix wasn't merged sooner.

cc: @Eveeifyeve

**Comment by Eveeifyeve** [comment]
Please let me know if there is anymore issues.

**Comment by Eveeifyeve** [comment]
> Please let me know if there is anymore issues arise.

As I will make a new pr that fixes more issues.
As it was my fault that it was broken.

**Comment by Eveeifyeve** [comment]
I feel like there is no more issues after having this open for 2 months.
cc: @OkelleyDevelopment close as completed?

**Comment by OkelleyDevelopment** [comment]
Good catch! Been busy the last few weekends so I hadn't checked back in 😅 


If anything comes up feel free to ping us!
